tests/schema/tests.py

```diff
@@ -136,6 +136,22 @@ def check_added_field_default(self, schema_editor, model, field, field_name, exp
                 database_default = cast_function(database_default)
             self.assertEqual(database_default, expected_default)
 
+    def count_constraints(self, table, column, to):
```

Comment:
A docstring might be helpful. In particular, I had to study the tests to figure out what value "to" is expected to take.